static/js/components/SourceDesktop.jsx

```diff
@@ -431,6 +433,13 @@ const SourceDesktop = ({ source }) => {
               </div>
             </div>
           )}
+          {source.summary_history?.length > 0 ? (
```

Comment:
There's a (good) chance that there's an AI summary in the DB for this, so try to show this component